src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -1102,40 +1102,28 @@ public void BuildAppWithManagedResourceParserAndLibraries ()
 		}
 
 		[Test]
-		[Category ("DotNetIgnore")] // n/a in .NET 5, not possible to use $(TFV) of v8.0
-		public void CheckMaxResWarningIsEmittedAsAWarning([Values (false, true)] bool useAapt2)
+		public void CheckMaxResWarningIsEmittedAsAWarning()
 		{
-			AssertAaptSupported (useAapt2);
 			var path = Path.Combine ("temp", TestName);
 			var proj = new XamarinAndroidApplicationProject () {
-				TargetFrameworkVersion = "v8.0",
-				TargetSdkVersion = "26",
+				TargetSdkVersion = "32",
```

Comment:
I think it's safe to remove this, it should be ignored since the default OS version in the TFM is now `34`? We should be able to remove `MinSdkVersion` and `UseLatestPlatformSdk` below for simplicity as well.